tests/api.spec.ts

```diff
@@ -266,7 +269,6 @@ describe('Requests to public API', () => {
       expect(response.value.status !== AnalysisStatus.failed).toBeTruthy();
 
       if (response.value.status === AnalysisStatus.complete) {
-        expect(response.value.sarif).toMatchSnapshot();
```

Comment:
we're removing those kind of tests because we don't want the flakiness of updating snapshots whenever the extensions list changes?

Reply:
yes